course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -957,6 +957,7 @@ def _add_course_instructors(self, course_instructors, course_run):
         """
         Create and add instructors to a course run.
         """
+        course_run.staff.clear()
```

Comment:
don't we need to update unit tests for this change?

Reply:
This is already being test in the API test.